app/controllers/internal/UpscanCallbackController.scala

```diff
@@ -45,9 +44,9 @@ class UpscanCallbackController @Inject()(mcc: MessagesControllerComponents,
 
   private[controllers] def deriveFileStatus(fileUpload: FileUpload): FileUpload = {
     fileUpload.failureDetails match {
-      case Some(details) if(details.failureReason=="QUARANTINE") =>
+      case Some(details) if (details.failureReason == "QUARANTINE") =>
```

Comment:
I think the round brackets here are redundant

Reply:
that's code formatting adding spaces and the if statement isn't my change really, I personally like having brackets though you're right they're not mandatory here